packages/react-data-grid/src/Cell.js

```diff
@@ -71,43 +71,21 @@ class Cell extends React.Component {
   }
 
   componentDidUpdate() {
-    this.checkFocus();
+
     let dragged = this.props.cellMetaData.dragged;
     if (dragged && dragged.complete === true) {
       this.props.cellMetaData.handleTerminateDrag();
     }
-    if (this.state.isCellValueChanging && this.props.selectedColumn != null) {
-      this.applyUpdateClass();
-    }
     if (this.state.isLockChanging && !this.props.column.locked) {
       this.removeScroll();
     }
   }
 
-  shouldComponentUpdate(nextProps: any): boolean {
-    let shouldUpdate = this.props.column.width !== nextProps.column.width
-      || this.props.column.left !== nextProps.column.left
-      || this.props.column.cellClass !== nextProps.column.cellClass
-      || this.props.height !== nextProps.height
-      || this.props.rowIdx !== nextProps.rowIdx
-      || this.isCellSelectionChanging(nextProps)
-      || this.isDraggedCellChanging(nextProps)
-      || this.isCopyCellChanging(nextProps)
-      || this.props.isRowSelected !== nextProps.isRowSelected
-      || this.isSelected()
-      || this.props.isCellValueChanging(this.props.value, nextProps.value)
-      || this.props.forceUpdate === true
-      || this.props.className !== nextProps.className
-      || this.props.expandableOptions !== nextProps.expandableOptions
-      || this.hasChangedDependentValues(nextProps)
-      || this.props.column.locked !== nextProps.column.locked;
-    return shouldUpdate;
-  }
-
   onCellClick = (e) => {
```

Comment:
using rxjs should allow us to reduce the complexity of our shouldComponentUpdate functions